src/mm/webclientservlets/GetAllPairs.java

```diff
@@ -45,13 +45,15 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
 		System.out.println("Get AllPairs Servlet");
 
 		ArrayList<Pair> pairsArray = new ArrayList<Pair>();
-		ArrayList<PairsInfo> pairsMainInfo = new ArrayList<PairsInfo>();

```

Comment:
@firdos94  @danielzar2710  doesn't the PairsInfo model have more info than Pair??

Reply:
no

Reply:
it's missing some importent data

Reply:
My version works when i get apairs now

Reply:
so if pair is enough, do we still need PairsInfo at all?